pkg/errmetrics/errmetrics.go

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package errmetrics
+
+import (
+	_ "embed"
+	"encoding/json"
+	"log"
+)
+
+var (
+	MapName = "errmetrics_map"
+)
+
+// json entry
+type Entry struct {
+	ID       int    `json:"id"`
+	Filename string `json:"filename"`
+}
+
+//go:embed fileids.json
+var fileIDsJSON []byte
+
+func initFileIDs() map[int]string {
+	var entries []Entry
+	if err := json.Unmarshal(fileIDsJSON, &entries); err != nil {
+		log.Panic(err)
+		return nil
+	}
+
+	ret := map[int]string{}
+	for _, e := range entries {
+		ret[e.ID] = e.Filename
+	}
+	return ret
+}
+
+var fileIDs = initFileIDs()
```

Comment:
why not exposing this as a function and return an error if the file is invalid? You want it to panic on load?

Reply:
OK, fixed to be prettier :)
(I didn't think it mattered since i) the file is static and ii) this is only used by tetra and the utility that generates ids, but it was easy to fix so fixed)